geonode/base/migrations/0059_fill_empty_resource_type.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.16 on 2021-05-03 09:40
+
+from django.db import migrations
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Q
+
+def set_resource_type(apps, _):
+    MyModel = apps.get_model('base', 'ResourceBase')
+    resources = MyModel.objects.filter(Q(resource_type=None) | Q(resource_type=""))
+    for resource in resources:
+        rtype = ContentType.objects.get(id=resource.polymorphic_ctype_id)
```

Comment:
If some exception occurs here for any reason we end up blocking the migrations at all.

Let's maybe encapsulate this on a try catch block on log any error accordingly.